packages/trader/src/Modules/Contract/Containers/contract.jsx

```diff
@@ -9,10 +9,20 @@ import { connect }       from 'Stores/connect';
 import ContractReplay    from './contract-replay.jsx';
 
 class Contract extends React.Component {
+    constructor () {
+        super();
+        this.redirectOnClick.bind(this);
```

Comment:
Question, are we binding `this` here to prevent loss of context in `strict` mode? 
Because the `redirectOnClick` is already a constructed function.